src/logics/openocean-v2/logic.swap-token.test.ts

```diff
@@ -31,7 +31,7 @@ describe('OpenOceanV2 SwapTokenLogic', function () {
       {
         chainId: common.ChainId.metis,
         fields: {
-          input: new common.TokenAmount(common.metisTokens['METIS(ERC20)'], '1'),
```

Comment:
pls also fix the test in test/logics/openocean-v2

Reply:
fixed